package/yast2-kdump.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Fri Dec 15 11:34:57 UTC 2017 - jreidinger@suse.com
+
+- Show kdump as disabled in proposal if product disable it
+  (bsc#1071896)
```

Comment:
"selecting a volume group on a disk without partitions causes ...",are you sure it's the right bug?

Reply:
Ah, my fault. Too much open bugs. Will fix it.